src/main/native/darwin/fsevents.cc

```diff
@@ -160,11 +162,15 @@ Java_com_google_devtools_build_lib_skyframe_MacOSXFsEventsDiffAwareness_poll(
   if (info->everything_changed) {
     result = nullptr;
   } else {
-    jclass classString = env->FindClass("java/lang/String");
-    result = env->NewObjectArray(info->paths.size(), classString, nullptr);
+    jclass classByteArray = env->FindClass("[B");
+    result = env->NewObjectArray(info->paths.size(), classByteArray, nullptr);
```

Comment:
Is this how you create an Object array containing byte arrays? It's been ages that I wrote JNI code, but naively I'd think that you'd need to put `java.lang.Object` here since the return value is `Object[]` and not `byte[][]`.